credentials/bc-registries-dbc-cred-doc/DBC-cred-doc-v1

```diff
@@ -0,0 +1,720 @@
+  </tr>
+  <tr>
+    <th>Data Type</th>
+    <td>String</td>
+  </tr>
+  <tr>
+    <th>Format</th>
+    <td>
+        <ul>
+            <li>Maximum 35 characters</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <th>Rules</th>
+    <td>
+        <ul>
+            <li>Never blank</li>
+            <li>Always upper case</li>
+            <li>Family names over 35 characters are truncated</li>
+            <li>The only characters allowed are the letters A through Z and the following "special characters": hyphen, apostrophe, period, and space</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <th>Notes</th>
+    <td>
+        <ul>
+            <li>ast Names with spaces or punctuation (e.g., “St. John”, “O’Brian”, “van Cleef”, “Scott-Bigsby”) will have the spaces and punctuation included if the individual has a photo BC Services Card but removed if the individual has a non-photo BC Services Card</li>
+            <li>A mononym may be duplicated in this attribute and the family_name unless it has a space in it, in which case the first part may be recorded in this attribute and the second part in the family_name attribute</li>
+        </ul>
+    </td>
+  </tr>
+</table>
+
+#### Role
+
+<table>
+  <tr>
+    <th>Attribute</th>
+    <td>role</td>
+  </tr>
+  <tr>
+    <th>Description</th>
+    <td>The person's role(s) with the business, if any. <br>
+    Roles are limited to those tracked by BC Registries, and do not normally correspond to job titles.</td>
+  </tr>
+  <tr>
+    <th>Source</th>
+    <td>Currently a business filing, from when an individual sets up their own proprietorship; during this process, they declare they are the proprietor. <br>
+    In the future:
+    <ul><li>Other roles will be determined through business filings supported by a process to authenticate and link a Registries Account record to the filing information</li></ul>
+    </td>
+  </tr>
+  <tr>
+    <th>Data Type</th>
+    <td>String</td>
+  </tr>
+  <tr>
+    <th>Format</th>
+    <td>
+        <ul>
+            <li>Maximum 30 characters</li>
+        </ul>
+    </td>
+  </tr>
+  <tr>
+    <th>Rules</th>
+    <td>
+        <ul>
+            <li>Roles are limited to those defined by BC Registries (and/or its governing legislation) and by the Business Type. (For clarity, this will not include the person's role within or in relation to the organization, e.g., CEO, Accountant.)</li>
+            <li>Allowable values (currently) are:</li>
+            <ul><li>Proprietor</li></ul>
+            <li>In future, allowable values may include:</li>
+            <ul>
+                <li>Incorporator</li>
+                <li>Partner</li>
+                <li>Director</li>
+                <li>An indicator (to be determined) that the individual does not have such a role, or if their role(s) cannot be confidently determined by BC Registries (see Notes)</li>
```

Comment:
" (see Notes)" should be removed, as there is not a notes section for this attribute.
This error was in the original document.